etc/patches/mariadb/compat.patch

```diff
@@ -31,13 +31,11 @@ index 0d4ddb9..22d0dcd 100644
 +
  config()
  {
--    # TokuDB requires third-party software jemalloci (see http://www.canonware.com/jemalloc/)
--    cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX} -DPLUGIN_TOKUDB=NO
-+    if [[ $(uname) == Linux ]]; then
-+        # TokuDB requires third-party software jemalloci (see http://www.canonware.com/jemalloc/)
-+        cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX} -DPLUGIN_TOKUDB=NO -DHAVE_CXX_NEW=NO -DPLUGIN_MROONGA=NO
-+    else
-+        # TokuDB requires third-party software jemalloci (see http://www.canonware.com/jemalloc/)
-+        cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX} -DPLUGIN_TOKUDB=NO
-+    fi
+     # Disable external SSL on OS X as it is not a standard library
+@@ -10,5 +13,5 @@ config()
+     fi
+ 
+     # TokuDB requires third-party software jemalloci (see http://www.canonware.com/jemalloc/)
+-    cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX} -DPLUGIN_TOKUDB=NO $ssloptions
++    cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX} -DHAVE_CXX_NEW=NO -DPLUGIN_MROONGA=NO -DPLUGIN_TOKUDB=NO $ssloptions
```

Comment:
That seems reasonable.  The main goal was to fix the conflict with the upstream changes.